poverty_info.rb

```diff
@@ -0,0 +1,47 @@
+require 'pry'
```

Comment:
Super nitpick but try to get in the hang of removing any remnants of your debugging process when you submit your PRs